src/main/java/com/meetme/plugins/jira/gerrit/data/IssueReviewsImpl.java

```diff
@@ -19,10 +19,7 @@
 import com.atlassian.jira.issue.Issue;
 import com.atlassian.jira.issue.IssueManager;
 import com.meetme.plugins.jira.gerrit.tabpanel.GerritEventKeys;
-import com.sonymobile.tools.gerrit.gerritevents.GerritQueryException;
-import com.sonymobile.tools.gerrit.gerritevents.GerritQueryHandlerHttp;
-import com.sonymobile.tools.gerrit.gerritevents.GerritQueryHandlerWithPersistedConnection;
-import com.sonymobile.tools.gerrit.gerritevents.QueryHandler;
+import com.sonymobile.tools.gerrit.gerritevents.*;
```

Comment:
Can we use single class import ? 
https://intellij-support.jetbrains.com/hc/en-us/community/posts/360000042464-Imports-getting-starred-in-spite-of-Use-single-class-import-option

Reply:
Yes, i missed that. Will fix it in a new commit

Reply:
Any news?